client/src/components/Followers.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const Followers = ({followers}) => (
```

Comment:
Do your destructuring inside the component. It's cleaner, and it also will allow you to do exception handling if the component doesn't receive the correct props.